rts/engine/game.cc

```diff
@@ -383,6 +383,9 @@ PlayerId RTSGame::MainLoop(const std::atomic_bool *done) {
           for (const auto &bot : _bots) {
               bot->Act(_env, true);
           }
+          if (_spectator != nullptr) {
+            _spectator->Act(_env);
+          }
```

Comment:
@yuandong-tian Without that, the last status/frame is never sent to the spectator, since the last status is calculated here. `_spector->Act` does nothing but send out the status of the game.

Visualization works fine without it. Yet minirts.html won't receive the last status of the game.  When a game is over, minirts.html shows a base with 5 hp left, which should be destroyed by the last move.